executor/executor_distsql.go

```diff
@@ -294,7 +302,28 @@ func closeAll(objs ...Closeable) error {
 	return nil
 }
 
-// XSelectIndexExec represents XAPI select index executor without result fields.
+// XSelectIndexExec represents DistSQL select index executor.
+// There are two execution modes. one is single-read, in which case we only need to read index keys.
+// Another one is double-read, we first do index request to get handles, we use each partial result to build
```

Comment:
-> The other one is double-read, in which case we first do index request to get handles, then we use each partial result to build